src/pages/AMM/Swap/Swap.tsx

```diff
@@ -1,7 +1,30 @@
+import { useState } from 'react';
+
+import { RELAY_CHAIN_NATIVE_TOKEN } from '@/config/relay-chains';
 import MainContainer from '@/parts/MainContainer';
+import { SwapPair } from '@/types/swap';
+
+import { SwapForm, SwapLiquidity } from './components';
+import { StyledWrapper } from './Swap.style';
 
 const Swap = (): JSX.Element => {
-  return <MainContainer>Swap</MainContainer>;
+  // const accountId = useAccountId();
+  const [pair, setPair] = useState<SwapPair>({ input: RELAY_CHAIN_NATIVE_TOKEN });
+
+  // if (isLoading) {
```

Comment:
Can this code be removed? Or is it only commented out temporarily?

Reply:
it is temporary